packages/jaeger-ui/package.json

```diff
@@ -2,44 +2,45 @@
   "private": true,
   "name": "jaeger-ui",
   "version": "1.27.3",
-  "main": "src/index.js",
+  "main": "src/index.tsx",
   "license": "Apache-2.0",
   "homepage": ".",
-  "workspaces": {
-    "nohoist": [
-      "customize-cra",
-      "customize-cra/**",
-      "react-scripts",
-      "react-scripts/**",
-      "react-app-rewired",
-      "react-app-rewired/**"
-    ]
-  },
   "repository": {
     "type": "git",
     "url": "https://github.com/jaegertracing/jaeger-ui.git",
     "directory": "packages/jaeger-ui"
   },
   "devDependencies": {
+    "@babel/core": "^7.21.0",
+    "@babel/preset-env": "^7.20.2",
+    "@babel/preset-react": "^7.18.6",
+    "@babel/preset-typescript": "^7.21.0",
+    "@svgr/babel-plugin-transform-svg-component": "^6.5.1",
+    "@svgr/babel-preset": "^6.5.1",
```

Comment:
@ mszabo-wikia do you know why/if these were needed?

Reply:
Yeah, this is likely an artifact. I thought it was still being used by tests for SVG support, but it seems those use `babel-plugin-inline-react-svg` instead (test/babel-transform.js). So these are probably OK to remove assuming that tests pass without them.

Reply:
thanks, appreciate your comments

Reply:
thanks for reaching out! :)